### PR TITLE
[windres-rc] support --define, --include-dir

### PR DESCRIPTION
**Pull request by wrobelda**
**Describe the pull request**
  Adds support for missing windres CLI switches:` --define` and  `--include`

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  Not applicable

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
 Not applicable

### CODE REVIEW

**Comment by JackBoosY** [comment]
I'm not sure about this part, needs @BillyONeal @ras0219-msft review this.

File: scripts/buildsystems/make_wrapper/windres-rc [hunk 1]
```diff
@@ -68,6 +68,16 @@ func_windres_wrapper ()
...
     else
       case $1 in
+	--define*)
+		eat=1
+		set x "$@" "-d $2"
+		shift
+		;;
+	--include-dir*)
+		eat=1
+		set x "$@" "-I $2"
```
Comment (Neumann-A) [inline scripts/buildsystems/make_wrapper/windres-rc:78]: since `$2` is a path doesn't this require `func_file_conv` ?

Comment (wrobelda) [reply scripts/buildsystems/make_wrapper/windres-rc]: Good call. Fixed.

**Comment by BillyONeal** [comment]
Related: https://github.com/microsoft/vcpkg/pull/26424

**Comment by JavierMatosD** [comment]
Thanks!

**Pull request merged**